Makefile

```diff
@@ -0,0 +1,5 @@
+build:
+	docker run -t -v "${PWD}":/root/berrymuch yamsergey/bb10-ndk:0.2 /bin/bash -c 'cd /root/berrymuch; ./build.sh -b /root/bbndk -t build'
```

Comment:
@yamsergey also need to add a command to build the container itself

Reply:
I would create a separated repo for that in the future. To be able to easily change/build/push the image. For now, I've created it in that way to test and debug.

Reply:
Ah I got it. I thought the image was meant to be built by the developer locally which is why the Dockerfile is there. I do believe that Dockerfile should live in a separate repo. Cool to keep it here today but I'd be itching to move it tomorrow. Creating a separate repo under BerryFarm is easy and the more repos we have the merrier :wink:

Reply:
@yamsergey in either case can you please update the Developer section of the README to refer to the new `make` workflow instead of the VM build?

Reply:
Sure, but first I want it to start work. So far I stuck on building `taskwarrior`. https://circleci.com/gh/BerryFarm/berrymuch/15

Reply:
For some reason CircleCI skips through some packages. For example, it doesn't build ca-certificates, git and curl - lynx follows right after openssl. Taskwarrior needs libuuid built first which apparently is also skipped.

Can you try running this snippet on CircleCI to see what sequence it assumes?

```
PORTSDIR="$PWD/ports"

ALLPORTS="bash bc bison bzip2 cflow coreutils cronie diffutils ed fakeroot file findutils gcc gdbm gettext grep groff gzip ircii jansson libevent libmpg123 libuuid m4 make man oniguruma jq openssh openssl ca-certificates curl git lynx patch rsync sc sqlite tar taskwarrior tmux vim wget xz yaml youtube-dl zeromq zlib ansiweather"
  BROKEN="gdb"

cd "$PORTSDIR"
for dir in $ALLPORTS
do
  if [ -d "$dir" ] && [ -e "$dir/build.sh" ]; then
    [ -f "$dir/vars.sh" ] && {
      . $dir/vars.sh
      [ -f ../packages/$DISTVER.zip ] && {
        echo "Warning: skipping build of $DISTVER (file packages/$DISTVER.zip already exists) "
        continue;
      }
    }
    echo "Building $dir"
    cd "$dir"
    cd "$PORTSDIR"
  fi
done

```

Reply:
It has skipped them because I wanted to save some credits there. As you can see the last build has been successful  👍 But we don't have build time on circle ci for this week any more :( I've opened PR with GitHub actions integration instead

Reply:
Great so I'm not going crazy. Please remove `uuid-dev`, adjust README and let's merge this baby already :smiley:

Reply:
👍